pkg/client/events_query/dialer.go

```diff
@@ -0,0 +1,35 @@
+package eventsquery
```

Comment:
Good call! :raised_hands: 

I went with moving `connection.go` and `dialer.go` into `pkg/client/websocket`. :+1: 

This resulted in also having to move `ErrReceive` into a new `pkg/client/websocket/errors.go` file to avoid an import loop.